docs/pages/split_stream.md

```diff
@@ -1,45 +1,86 @@
-# Splitting the eventstream
+# Split Stream
 
-In some cases the business has rules which implies an restart of the event stream for an aggregate since the past events
-are not relevant for the current state. For example a user decides to end his active subscription and the business rules
-says if the user start a new subscription all past events should not be considered anymore. Another case could be a
-banking scenario. There the business decides to save the current state every quarter for each banking account.
+In some cases the business has rules which implies a restart of the event stream for an aggregate
+since the past events are not relevant for the current state.
+A bank is often used as an example. A bank account has hundreds of transactions,
+but every bank makes a balance report at the end of the year.
+In this step the current account balance is persisted.
+This event is perfect to split the stream and start aggregating from this point.
 
-Not only that some businesses requires such an action it also increases the performance for aggregate which would have a
-really long event stream.
+Not only that some businesses requires such an action
+it also increases the performance for aggregate which would have a really long event stream.
 
-## Flagging an event to split the stream
+In the background the library will mark all past events as archived
+and will not load them anymore for building the aggregate.
+It will only load the events from the split event and onwards.
+But subscriptions will still receive all events.
+So you can create projections which are based on the full event stream.
 
-To use this feature you need to add the `SplitStreamDecorator`. You will also need events which will trigger this
-action. For that you can use the `#[SplitStream]` attribute. We decided that we are not literallty splitting the stream,
-instead we are marking all past events as archived as soon as this event is saved. Then the past events will not be
-loaded anymore for building the aggregate. This means that all needed data has to be present in these events which
-should trigger the event split.
+!!! tip
+
+    To split the stream is often a better solution than using snapshots.
```

Comment:
I'm not sure if this statement is really correct. Performance wise Snapshotting is better - but not related to any business requirements.

Reply:
removed